packages/drivers/routerlicious-driver/src/retriableGitManager.ts

```diff
@@ -140,6 +140,13 @@ export class RetriableGitManager implements IGitManager {
         );
     }
 
+    public async deleteSummary(softDelete: boolean): Promise<void> {
+        return this.runWithRetry(
+            async () => this.internalGitManager.deleteSummary(softDelete),
+            "gitManager_deleteSummary",
+        );
+    }
```

Comment:
@TriDucNgo , updating the version picked up your breaking change. Does this look correct to you?

Reply:
Thanks @scarlettjlee, that looks correct to me.